src/shared/utils/browser/clear-auth-cookie.ts

```diff
@@ -0,0 +1,12 @@
+import * as cookie from "cookie";
+import { authCookieName } from "shared/config";
+
+export default function clearAuthCookie() {
+  document.cookie = cookie.serialize(authCookieName, "", {
+    expires: new Date("Thu, 01 Jan 1970 00:00:01 GMT"),
+    maxAge: -1,
```

Comment:
Changed from `0` to `-1`.

Reply:
What is the point of having both an `expires` and a `maxAge`?

Reply:
I found [something relevant](https://mrcoles.com/blog/cookies-max-age-vs-expires/):

> **Q. What if I set both expires and max-age in a cookie?**
A. Every browser that supports max-age will ignore the expires regardless of it’s value, and likewise, Internet Explorer will ignore the max-age and just use expires.

Reply:
Nice, I've removed `expires`. Good call.